server/bundler/loader.js

```diff
@@ -33,6 +35,11 @@ function getSectionsModule( sections ) {
 		'	},',
 		'	load: function() {',
 		'		' + sectionLoaders,
+		'	},',
+		'	preload: function( section ) {',
+		'		switch ( section ) {',
```

Comment:
Why do we need the `switch` statement here? I was thinking `sections.preload()` would just `require.ensure` a specific module.